src/discussions/posts/post-editor/PostEditor.jsx

```diff
@@ -281,24 +281,24 @@ function PostEditor({
                 ))}
               </Form.Control>
             </Form.Group>
-            {canSelectCohort(values.topic)
-              && (
-                <Form.Group className="w-50 d-inline-block pl-2">
-                  <Form.Control
-                    name="cohort"
-                    as="select"
-                    value={values.cohort}
-                    onChange={handleChange}
-                    onBlur={handleBlur}
-                    floatingLabel={intl.formatMessage(messages.cohortVisibility)}
-                  >
-                    <option value="">{intl.formatMessage(messages.cohortVisibilityAllLearners)}</option>
-                    {cohorts.map(cohort => (
-                      <option key={cohort.id} value={cohort.id}>{cohort.name}</option>
-                    ))}
-                  </Form.Control>
-                </Form.Group>
-              )}
+            {canSelectCohort(values.topic) && (
+            <Form.Group className="w-50 d-inline-block pl-2">
+              <Form.Control
+                name="cohort"
+                as="select"
+                value={values.cohort}
+                onChange={handleChange}
+                onBlur={handleBlur}
+                aria-describedby="cohortAreaInput"
+                floatingLabel={intl.formatMessage(messages.cohortVisibility)}
+              >
+                <option value="default">{intl.formatMessage(messages.cohortVisibilityAllLearners)}</option>
```

Comment:
In case of all learners selected the value sent to the backend will be "default"

Reply:
No, a Null value will be sent in the case of all learners selected.